src/V3EmitC.cpp

```diff
@@ -2438,9 +2449,9 @@ void EmitCStmts::emitOpName(AstNode* nodep, const string& format, AstNode* lhsp,
                     needComma = true;
                     break;
                 case 'W':
-                    if (lhsp->isWide()) {
```

Comment:
Does this not break other operators which should have `lhsp` here?

Reply:
It should work, cause before we are using this pointer we are choosing the right node